### DISCUSSION TITLE
New project no loaders crashing

### DISCUSSION THREAD

**Discussion by SpicyPete** [vercel/next.js#13098] [Help]
I'm working on a new NextJS project, and it works great on my MacBook, but on my windows computer I'm getting the following error.  

```
Module parse failed: Unexpected token (5:4)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
```  

The reproduction steps are simple.  

`yarn create next-app`
`-> select default`
`cd [project-name]`
`yarn dev`
`> navigate localhost:3000`
`Error`  

I've spent probably 2-3 hours trying to solve this, and am going a bit crazy.   Any insight and suggestions would be appreciated.  

Running Node v12.13.1

**Comment by jamesmosier** [comment]
Could you share more of the error output?
